src/lib/style-formatting/index.js

```diff
@@ -114,7 +114,7 @@ function handleExplode(value, parameter) {
     });
   }
 
-  if (typeof value === 'object') {
+  if (typeof value === 'object' && value !== null) {
```

Comment:
The amount of times this bug has hit me... probably the best advertisement for migrating to Typescript.